aws/policy/networking.yaml

```diff
@@ -48,6 +48,7 @@ Statement:
       - ec2:DeleteRoute
       - ec2:AssociateRouteTable
       - ec2:DisassociateRouteTable
+      - ec2:ReplaceRouteTableAssociation
```

Comment:
I don't think we actually have a module that does this.

Reply:
Yeah, it was in the 'hacking' policy and since we allow associate and disassociate it seemed reasonable to move it into the main list, I can remove it if you'd rather.

Reply:
Yeah, I saw why you did it. I'd personally prefer to keep the CI policies minimal.